data/img/material/white/apps.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#FFFFFF"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M4 8h4V4H4v4zm6 12h4v-4h-4v4zm-6 0h4v-4H4v4zm0-6h4v-4H4v4zm6 0h4v-4h-4v4zm6-10v4h4V4h-4zm-6 4h4V4h-4v4zm6 6h4v-4h-4v4zm0 6h4v-4h-4v4z"/></svg>
```

Comment:
Did you create all the SVG's yourself? I just want to make sure they are properly licensed.

Reply:
Please excuse, I should have mentioned it. Since flameshot is already using "Google Material Design Icons", I took the new icons from there as well:
- https://fonts.google.com/icons?selected=Material+Icons+Round:apps:
- https://fonts.google.com/icons?selected=Material+Icons+Round:image: